packages/tasit-sdk/src/helpers/DecentralandUtils.test.js

```diff
@@ -0,0 +1,81 @@
+import { Action } from "../TasitSdk";
+const { ConfigLoader, ERC20, ERC721, Marketplace } = Action;
+const { Mana } = ERC20;
+const { Estate, Land } = ERC721;
+const { Decentraland } = Marketplace;
+import DecentralandUtils from "./DecentralandUtils";
+
+// Note: This test suite assumes that all parcels and estates of SellerWallet are for sale
+// See more: `TasitSDk/packages/tasit-contracts/src/script/populateDecentralandContracts.js`
+describe("DecentralandUtils", () => {
+  const [minterWallet, sellerWallet] = accounts;
+  const { address: sellerAddress } = sellerWallet;
+  let mana;
+  let land;
+  let estate;
+  let marketplace;
+  const decentralandUtils = new DecentralandUtils();
+  const {
+    getAllAssetsForSale,
+    getEstateIdsOf,
+    getParcelIdsOf,
+    getAssetsOf,
+  } = decentralandUtils;
+  let assetsForSale;
+
+  const isAssetAnEstate = asset => asset.nftAddress === estate.getAddress();
+  const isAssetAParcel = asset => asset.nftAddress === land.getAddress();
+
+  beforeEach("", async () => {
+    mana = new Mana(MANA_ADDRESS);
+    land = new Land(LAND_PROXY_ADDRESS);
+    estate = new Estate(ESTATE_ADDRESS);
+    marketplace = new Decentraland(MARKETPLACE_ADDRESS);
+
+    assetsForSale = await getAllAssetsForSale();
```

Comment:
Just because something will be reused in each test doesn't necessarily mean it goes in the beforeEach, from my perspective. For something that isn't really "test setup" but is actually something that would need to happen during the code flow in the real-world version of what is being tested, I'd almost like to put this in each function, and if it became multiple lines, put it in a helper function that is called in each test. What do you think? This is a weakly held opinion of mine, FYI, so feel free to push back.

Reply:
I agree with you. I've put it there because it's a read-only test suite and for saving a few (milli)seconds.

Reply:
For me, we have good motivations to keep or move to each test case. Your call.

Reply:
No strong preference - mostly just noting this for the future precedent we may want to set. We can keep it as-is here for now.

Reply:
Ok, I'll move to the test cases since the time saved is < 1sec.

Reply:
Done.

Reply:
Okay, thanks! Resolving this.